app/_src/networking/transparent-proxy/reachable-backends.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Reachable Backends
+content_type: how-to
+---
+
+{% assign docs = "/docs/" | append: page.release %}
+{% assign Kuma = site.mesh_product_name %}
+{% assign tproxy = site.data.tproxy %}
+
+{% warning %}
+This feature works only when [MeshService]({{ docs }}/networking/meshservice) is enabled.
+{% endwarning %}
+
+Reachable Backends provides similar functionality to [Reachable Services]({{ docs }}/networking/transparent-proxy/reachable-services/), but it applies to resources such as [MeshService]({{ docs }}/networking/meshservice), [MeshExternalService]({{ docs }}/networking/meshexternalservice), and [MeshMultiZoneService]({{ docs }}/networking/meshmultizoneservice).
+
+By default, each data plane proxy tracks all other data planes in the mesh, which can impact performance and use more resources. Configuring `reachableBackends` allows you to specify only the services your application actually needs to communicate with, improving efficiency.
+
+Unlike Reachable Services, Reachable Backends uses a structured model to define the resources.
+
+### Model
+
+- **refs**: Lists the resources your application needs to connect with, including:
+  - **kind**: Type of resource. Options include:
+    - **MeshService**
+    - **MeshExternalService**
+    - **MeshMultiZoneService**
+  - **name**: Name of the resource.
+  - **namespace**: (Kubernetes only) Namespace where the resource is located. Required if using `namespace`.
+  - **labels**: A list of labels to match resources. You can define either `labels` or `name`.
```

Comment:
It's worth mentioning here that when we use this it matches ALL the workloads that have these labels. I think that's how reachableBackends is so much better than reachableServices